technic_worldgen/nodes.lua

```diff
@@ -1,152 +1,203 @@
+	groups = {cracky=3, pickaxey=1},
+	_mcl_hardness = 0.8,
+	_mcl_blast_resistance = 1,
+	sounds = technic.sounds.node_sound_stone_defaults(),
 	drop = "technic:zinc_lump",
 })
 
 minetest.register_node(":technic:mineral_lead", {
 	description = S("Lead Ore"),
 	tiles = {"default_stone.png^technic_mineral_lead.png"},
 	is_ground_content = true,
-	groups = {cracky=3},
-	sounds = default.node_sound_stone_defaults(),
+	groups = {cracky=3, pickaxey=1},
+	_mcl_hardness = 0.8,
+	_mcl_blast_resistance = 1,
+	sounds = technic.sounds.node_sound_stone_defaults(),
 	drop = "technic:lead_lump",
 })
 
 minetest.register_node(":technic:mineral_sulfur", {
 	description = S("Sulfur Ore"),
 	tiles = {"default_stone.png^technic_mineral_sulfur.png"},
 	is_ground_content = true,
-	groups = {cracky=3},
-	sounds = default.node_sound_stone_defaults(),
+	groups = {cracky=3, pickaxey=1},
+	_mcl_hardness = 0.8,
+	_mcl_blast_resistance = 1,
+	sounds = technic.sounds.node_sound_stone_defaults(),
 	drop = "technic:sulfur_lump",
 })
 
-minetest.register_node(":technic:granite", {
+if has_mcl then
+   minetest.register_alias("technic:granite", "mcl_core:granite")
+   minetest.register_alias("technic:granite_bricks", "mcl_core:granite_smooth")
+else
+   minetest.register_node(":technic:granite", {
 	description = S("Granite"),
 	tiles = {"technic_granite.png"},
 	is_ground_content = true,
 	groups = {cracky=1},
-	sounds = default.node_sound_stone_defaults(),
-})
+	sounds = technic.sounds.node_sound_stone_defaults(),
+   })
 
-minetest.register_node(":technic:granite_bricks", {
+   minetest.register_node(":technic:granite_bricks", {
 	description = S("Granite Bricks"),
 	tiles = {"technic_granite_bricks.png"},
 	is_ground_content = true,
 	groups = {cracky=1},
-	sounds = default.node_sound_stone_defaults(),
-})
+	sounds = technic.sounds.node_sound_stone_defaults(),
+   })
+end
 
 minetest.register_node(":technic:marble", {
 	description = S("Marble"),
 	tiles = {"technic_marble.png"},
 	is_ground_content = true,
-	groups = {cracky=3, marble=1},
-	sounds = default.node_sound_stone_defaults(),
+	groups = {cracky=3, marble=1, pickaxey=1},
+	_mcl_hardness = 0.8,
+	_mcl_blast_resistance = 1,
+	sounds = technic.sounds.node_sound_stone_defaults(),
 })
 
 minetest.register_node(":technic:marble_bricks", {
 	description = S("Marble Bricks"),
 	tiles = {"technic_marble_bricks.png"},
 	is_ground_content = true,
-	groups = {cracky=3},
-	sounds = default.node_sound_stone_defaults(),
+	groups = {cracky=3, pickaxey=1},
+	_mcl_hardness = 0.8,
+	_mcl_blast_resistance = 1,
+	sounds = technic.sounds.node_sound_stone_defaults(),
 })
 
 minetest.register_node(":technic:uranium_block", {
 	description = S("Uranium Block"),
 	tiles = {"technic_uranium_block.png"},
 	is_ground_content = true,
-	groups = {uranium_block=1, cracky=1, level=2, radioactive=2},
-	sounds = default.node_sound_stone_defaults()
+	groups = {uranium_block=1, cracky=1, level=dig_level, radioactive=2, pickaxey=4},
```

Comment:
Could maybe be better to explicitly select level group for each registration instead of defining and reassigning variable for it.

Reasoning: reassigning variable somewhere in between long list is likely to cause some mistakes later on and is harder to read because you have to keep track of that variable value instead of seeing it right there.

I think selecting values inline would be best straightforward thing to do without having to add registration wrappers, I mean basically this with `dig_level` variable removed altogether:
```suggestion
	groups = {uranium_block=1, cracky=1, level=has_mcl and 0 or 2, radioactive=2, pickaxey=4},
```

Reply:
done